src/org/opensolaris/opengrok/history/HistoryGuru.java

```diff
@@ -73,6 +73,8 @@
 
     private final int scanningDepth;
 
+    private String ignoreRepositoryName = ".opengrok_skip_history";
```

Comment:
I can certainly add that, but your later comments about not using .opengrok_skip_history at all warrants a complete rethink of the whole mechanism.